# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../acx100/eth-naming.patch
# Copyright (C) 2006 The OpenSDE Project
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./pci.c.orig	2006-05-20 15:11:13.000000000 -0400
+++ ./pci.c	2006-10-06 17:17:59.000000000 -0400
@@ -1538,7 +1538,7 @@
 		goto fail_irq;
 	}
 
-	ndev = alloc_netdev(sizeof(*adev), "wlan%d", dummy_netdev_init);
+	ndev = alloc_netdev(sizeof(*adev), "eth%d", dummy_netdev_init);
 	/* (NB: memsets to 0 entire area) */
 	if (!ndev) {
 		printk("acx: no memory for netdevice structure\n");
--- ./usb.c	2006-10-06 16:26:39.000000000 -0400
+++ ./usb.c.orig	2006-10-06 17:17:34.000000000 -0400
@@ -826,7 +826,7 @@
 
 	/* Allocate memory for a network device */
 
-	ndev = alloc_netdev(sizeof(*adev), "wlan%d", dummy_netdev_init);
+	ndev = alloc_netdev(sizeof(*adev), "eth%d", dummy_netdev_init);
 	/* (NB: memsets to 0 entire area) */
 	if (!ndev) {
 		msg = "acx: no memory for netdev\n";
